src/owl/dense/owl_dense_matrix_c.mli

```diff
@@ -411,14 +411,17 @@ val of_cols : mat array -> mat
 
 val print : ?max_row:int -> ?max_col:int -> ?header:bool -> ?fmt:(elt -> string) -> mat -> unit
 
-val save : mat -> string -> unit
+val save : mat -> out:string -> unit
```

Comment:
These are breaking changes. Are we sure we want to specify out as a labelled argument? I am not sure it is work that the breakage